src/bin/leftwm-check.rs

```diff
@@ -246,10 +222,7 @@ fn check_elogind(verbose: bool) -> Result<()> {
             }
             println!("\x1b[1;91mERROR: XDG_RUNTIME_DIR not set and elogind not found.\nSee https://github.com/leftwm/leftwm/wiki/XDG_RUNTIME_DIR for more information.\x1b[0m",);
```

Comment:
I probably left it because it's not part of the error instance itself... :thinking: I don't see any harm to remove it (or merge it with the error message below), it was probably a duplicate message before.

Reply:
Yeah I guess the link would be helpful to be merged, the rest is kind of duped

Reply:
Yeah, would be good to merge with the link I think

Reply:
Done in 91b5069